packages/api/src/controllers/webhook-log.ts

```diff
@@ -26,6 +26,7 @@ const requestsFieldsMap: FieldsMap = {
   userId: `webhook_log.data->>'userId'`,
   event: `webhook_log.data->>'event'`,
   statusCode: `webhook_log.data->'response'->>'status'`,
+  success: `webhook_log.data->>'success'`,
```

Comment:
I'm not sure how simple it would be, but you can consider adding unit test for this case.